lib/classification_lifecycle.rb

```diff
@@ -14,6 +14,12 @@ def queue(action)
       raise ClassificationNotPersisted.new(message)
     end
     ClassificationWorker.perform_async(classification.id, action)
+    if classification.complete?
+      classification.subject_ids.each do |sid|
+        ClassificationCountWorker.perform_async(sid, classification.workflow_id)
+        RetirementWorker.perform_async(sid, classification.workflow_id)
```

Comment:
I wanted them to get going at the same time to make sure subjects were retired as fast as possible. What if the RetirementWorker compares classification_count + 1?